bin/aca_py.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+
+
+def find_python_executable():
+    if "PYTHON" in os.environ:
+        return os.environ["PYTHON"]
+    python_executable = subprocess.getoutput("command -v python3")
+    if not python_executable:
+        python_executable = "python"
+    return python_executable
+
+
+def main():
+    python_executable = find_python_executable()
+    aries_cloudagent_args = sys.argv[1:]
+    subprocess.run(
+        [python_executable, "-m", "aries_cloudagent"] + aries_cloudagent_args
+    )
```

Comment:
This might actually solve the issue in my previous comment about multiple long-lived processes

Reply:
I agree. Wasn't aware of all the use cases for the original script so I figured it was best to stick as close to the original as possible. I will just access the module manually rather than calling the executable. Thanks for the feedback